sklearn/metrics/tests/test_ranking.py

```diff
@@ -30,6 +31,7 @@
 from sklearn.metrics import label_ranking_loss
 from sklearn.metrics import roc_auc_score
 from sklearn.metrics import roc_curve
+from sklearn.metrics.ranking import _ndcg_sample_scores, _dcg_sample_scores
```

Comment:
Would it be possible to tests the public (score-averaged) functions in addition to the private ones? They are tested in common tests (with respect to symmetry invariance etc) but still there is currently no tests verifying that `ndcg_score` and `dcg_score` produce the right values.